eWonConnector-gateway/src/main/java/org/imdc/ewon/SyncManager.java

```diff
@@ -267,9 +267,8 @@ protected void updateLive() {
                     try {
                         // Value is a string
                         if (valueString.charAt(0) == '\"') {
-                            // Strip the "s from the string
-                            // TODO: Consider the case where the string value has "s
-                            value = valueString.replaceAll("\"", "");
+                            // Strip leading and trailing double quote chars
```

Comment:
Is there any case where the first character may be a quote, but there is not a quote as the last character?

Reply:
no, quotes can only occur in strings which by definition in the m2web api start and end with "

Reply:
ok, perfect